docs/index.md

```diff
@@ -1,33 +1,30 @@
 # Start Here
 
-This is the {{company}} **Beta** documentation web site.
+This is the {{company}} documentation web site.
+The content you find here constitutes authoritative documentation for {{company}} products and services, including {{brand}} and the {{gui}}.
 
-## What does "Beta" mean?
+## Open source
 
-The fact that this site is in a Beta stage means that any information
-you find here should be reliable and accurate for {{company}}
-products and services. If you find any published documentation that is
-inaccurate or not in line with functionality as you observe it on
-{{brand}}, we would very much appreciate if you filed
-[a documentation bug]({{config.repo_url}}/issues).
+All content on this site is licensed under the [{{legal_docs.cc_by_sa.short_name}}]({{legal_docs.cc_by_sa.url}}) license.
 
-However, the documentation on this site may be incomplete, meaning
-that it does not yet cover *all* functionality available in
-{{brand}}.
+We host our site content [on GitHub]({{config.repo_url}}), and if you would like to help by contributing a patch, [here’s how](contrib/index.md)!
```

Comment:
```suggestion
We host our site content [on GitHub]({{config.repo_url}}), and if you would like to help by contributing a patch, [here’s how you can do it](contrib/index.md)!
```

Reply:
Hmmm. I'm not sure if that's an improvement? :)

"..., here's how you do that!" would be a change I could settle for.

Reply:
I think a verb is missing, hence the "do". And yes, "that" is better than "it".

Reply:
Done.